### PR TITLE
sql statement: refined Synopsis

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
    - https://github.com/pingcap/docs/pull/17045
    - https://github.com/pingcap/docs/pull/17047
    - https://github.com/pingcap/docs/pull/17049
    - https://github.com/pingcap/docs/pull/17051
    - https://github.com/pingcap/docs/pull/17052
    - https://github.com/pingcap/docs/pull/17053
    - https://github.com/pingcap/docs/pull/17065
    - https://github.com/pingcap/docs/pull/17068

- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-show-stats-histograms.md [hunk 1]
```diff
@@ -6,28 +6,23 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-histograms/']
 
 # SHOW STATS_HISTOGRAMS
 
-你可以使用 `SHOW STATS_HISTOGRAMS` 语句查看统计信息中直方图的相关信息。
+你可以使用 `SHOW STATS_HISTOGRAMS` 语句查看由 [`ANALYZE` statement](/sql-statements/sql-statement-analyze-table.md) 收集的直方图的相关信息，作为数据库 [statistics](/statistics.md)的一部分。
```
Comment (hfxsd) [inline sql-statements/sql-statement-show-stats-histograms.md:9]:
```suggestion
你可以使用 `SHOW STATS_HISTOGRAMS` 语句查看由 [`ANALYZE` statement](/sql-statements/sql-statement-analyze-table.md) 收集的直方图的相关信息，作为数据库 [statistics](/statistics.md) 的一部分。
```

File: sql-statements/sql-statement-show-stats-histograms.md [hunk 2]
```diff
@@ -6,28 +6,23 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-histograms/']
 
 # SHOW STATS_HISTOGRAMS
 
-你可以使用 `SHOW STATS_HISTOGRAMS` 语句查看统计信息中直方图的相关信息。
+你可以使用 `SHOW STATS_HISTOGRAMS` 语句查看由 [`ANALYZE` statement](/sql-statements/sql-statement-analyze-table.md) 收集的直方图的相关信息，作为数据库 [statistics](/statistics.md) 的一部分。
```
Comment (hfxsd) [inline sql-statements/sql-statement-show-stats-histograms.md:9]:
```suggestion
你可以使用 `SHOW STATS_HISTOGRAMS` 语句查看由 [`ANALYZE` statement](/sql-statements/sql-statement-analyze-table.md) 收集的直方图的相关信息，该内容是数据库 [statistics](/statistics.md) 的一部分。
```

File: user-defined-variables.md [hunk 3]
```diff
@@ -18,38 +18,28 @@ aliases: ['/docs-cn/dev/user-defined-variables/','/docs-cn/dev/reference/sql/lan
 
 ## 设置用户自定义变量
 
-用 `SET` 语句可以设置用户自定义变量，语法为 `SET @var_name = expr [, @var_name = expr] ...;`。例如：
-
-{{< copyable "sql" >}}
+用 [`SET` 语句](/sql-statements/sql-statement-set-variable.md) 可以设置用户自定义变量，语法为 `SET @var_name = expr [, @var_name = expr] ...;`。例如：
```
Comment (hfxsd) [inline user-defined-variables.md:21]:
```suggestion
用 [`SET` 语句](/sql-statements/sql-statement-set-variable.md)可以设置用户自定义变量，语法为 `SET @var_name = expr [, @var_name = expr] ...;`。例如：
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**